linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlinkat/rule.yml

```diff
@@ -27,7 +27,7 @@ description: |-
     Without generating audit records specific to the security and mission needs of the organization, it would be 
     difficult to establish, correlate, and investigate the events relating to an incident or identify those responsible for one.
     Audit records can be generated from various components within the information system (e.g., module or policy filter).
-    Add or update the following lines to <tt>/etc/audit/rules.d/audit.rules</tt> to configure the the operating system to generate 
+    Add or update the following lines to <tt>/etc/audit/rules.d/audit.rules</tt> to configure the operating system to generate 
```

Comment:
```suggestion
    Add or update the following lines to <tt>/etc/audit/rules.d/audit.rules</tt> to configure the {{{ full_name }}} to generate 
```

Reply:
I believe we should do a full evaluation of replacing `operating system` with `{{{ full_name }}}` within `rule.yml` files instead of doing this small changes. It may end up inconsistent and may not be necessary everywhere. I'm going to create an issue to track this.